charts/core/values.yaml.LB_wCSBA

```diff
@@ -4,8 +4,8 @@
 
```

Comment:
We should delete these files - I forgot to do so...

Reply:
Done, better?